installer/io.plotjuggler.application/meta/installscript.qs

```diff
@@ -101,4 +101,4 @@ Component.prototype.componentSelectionPageEntered = function()
     if (installer.fileExists(dir) && installer.fileExists(dir + "/maintenancetool.exe")) {
         installer.execute(dir + "/maintenancetool.exe", ["purge", "-c"]);
     }
-}
```

Comment:
can you also make sure this file isn't touched by this pr?

Reply:
pre-commit correction also.